userguide/support.rst

```diff
@@ -197,9 +188,9 @@ A series of instructional videos are available for %brand%:
 Professional Support
 --------------------
 
-In addition to the freely available community resources, professional
-support may be available through iXsystem's network of third-party
-consultants. Submit a support inquiry using the form at
+In addition to the free community resources, professional
+support is available through the third-party consultants of iXsystems.
```

Comment:
How about "support might be available in your area through third-party consultants."?